flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/serde/LogicalTypeJsonSerdeTest.java

```diff
@@ -249,28 +331,44 @@ public void testLogicalTypeSerde() throws IOException {
                                         ObjectIdentifier.of("cat", "db", "distinctType"),
                                         new VarCharType(false, 5))
                                 .build(),
+                        // custom RawType
+                        new RawType<>(Integer.class, IntSerializer.INSTANCE),
```

Comment:
Can you add a test case here for a "real" raw type? For example take `RAW(LocalDateTime.class, LocalDateTimeSerializer.INSTANCE)` (I use it in `CastRulesTest`)

Reply:
It doesn't really make a difference. But I replaced it with `LocalDateTime`.